### PR TITLE
Aastha/nested recycler view

### PR DESCRIPTION
**Pull request by aasthashah999**
This view shows all the lists a user has! Some things that I need to add are adding a see all button which will lead to the activity Cindy created for a single list! I will do this shortly.

### CODE REVIEW

File: app/src/main/java/com/example/flick/ListActivity.kt [hunk 1]
```diff
@@ -70,6 +68,29 @@ class ListActivity: AppCompatActivity() {
         Media(4, "Glee", "https://image.tmdb.org/t/p/w600_and_h900_bestv2/uW91bKQhdkwPLAE2lJJkY5xdyJg.jpg", listOf("director"), true, Date(2020), "very cool","KOR", "1 season", "very fool", listOf(), 1, "PG-13","5", "5", "5", listOf("Netflix"), listOf(), listOf() )
     )
 
+    private val listMediaShowVersion = listOf(
```
Comment (cindyh25) [inline app/src/main/java/com/example/flick/ListActivity.kt:71]: thanks for making all these changes for the list activity 😄

File: app/build.gradle [hunk 2]
```diff
@@ -45,4 +54,5 @@ dependencies {
     androidTestImplementation 'androidx.test.ext:junit:1.1.1'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
     implementation 'com.squareup.picasso:picasso:2.71828'
+    implementation 'com.github.bumptech.glide:glide:4.9.0'
```
Comment (cindyh25) [inline app/build.gradle:57]: did you end up using glide?

File: app/src/main/java/com/example/flick/ListActivity.kt (hunk 1 above)
Comment (aasthashah999) [reply app/src/main/java/com/example/flick/ListActivity.kt:71]: No problem! It looks really good now that the button to transition to the activity is there!

File: app/build.gradle (hunk 2 above)
Comment (aasthashah999) [reply app/build.gradle:57]: I did not :(. It did not end up working, I will have to remove that dependency if you are not using it!

**Pull request merged**